.docker/keycloak/Dockerfile.test

```diff
@@ -0,0 +1,7 @@
+FROM jboss/keycloak:11.0.2
```

Comment:
You might want to drop this to the latest v9 rather than v11 so it matches the gov keycloak version. They are using RedHat SSO 7.4 which is KC 9.0.11 under the hood: https://access.redhat.com/articles/2342881